cypher/cypher-docs/src/docs/dev/ql/administration/security/database/deny-database-access-syntax.asciidoc

```diff
@@ -1,7 +0,0 @@
-.Command syntax
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/database-administration.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/database-administration/#access-control-database-administration-access